django_pgjsonb/fields.py

```diff
@@ -174,8 +173,8 @@ def _alter_field(editor, model, old_field, new_field, old_type, new_type,
             for index_name in to_delete_indexes:
                 try:
                     editor.execute(editor._delete_constraint_sql(editor.sql_delete_index, model, index_name))
-                except ProgrammingError as exc:
-                    logger.Warning(exc)
+                except Exception as exc:
```

Comment:
I don't think capture any exception here is a good idea, since the only error we want to ignore here is the index has been deleted. Can you fix it?